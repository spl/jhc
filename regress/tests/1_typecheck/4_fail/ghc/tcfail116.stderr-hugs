ERROR "tcfail116.hs":6 - Ambiguous type signature in class declaration
*** ambiguous type : Foo a => ()
*** assigned to    : bug
