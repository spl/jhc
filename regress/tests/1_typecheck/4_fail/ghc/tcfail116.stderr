
tcfail116.hs:5:1:
    The class method `bug'
    mentions none of the type variables of the class Foo a
    When checking the class method: bug :: ()
    In the class declaration for `Foo'
